Magento_Theme/templates/html/copyright.phtml

```diff
@@ -0,0 +1,5 @@
+<div class="copyright">
+    <div class="copyright__content">
```

Comment:
class `copyright__content` is useless, you do not have to define this class if there is no styles for it. IMO there is no existing component you can use but maybe instead of creating `copyright` component we can create some `bar` component - naming more from function on the website not for function in the project. and this bar component, with some modifier if needed we will use later to the other 100% width bars